### PR TITLE
check in the public interface

### PR DESCRIPTION
**Pull request by nerdyyatrice**
This includes the public interfaces of aliyun fc java runtime

### CODE REVIEW

**Comment by rockuw** [comment]
LGTM.

Ping @shuaichang for cross check.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    public void handleRequest(InputStream input, OutputStream output, Context context) throws IOException {
+        FunctionComputeLogger logger = context.getLogger();
+        logger.debug(String.format("Handle request %s %n", context.getRequestId()));
+        byte[] data = new byte[1024];
+        output.write(context.getRequestId().getBytes());
+        output.write(System.lineSeparator().getBytes());
+        int nRead;
+        while ((nRead = input.read(data, 0, data.length)) != -1) {
+            output.write(data, 0, nRead);
+            logger.debug(String.format("handler received data %s %n", new String(data)));
```
Comment (shuaichang) [inline README.md]: You meant \n?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import java.io.InputStream;
+import java.io.OutputStream;
+
+/**
+ * This handler just echo the input with request ID
+ */
+public class EchoStreamHandler implements StreamRequestHandler {
+    public void handleRequest(InputStream input, OutputStream output, Context context) throws IOException {
+        FunctionComputeLogger logger = context.getLogger();
+        logger.debug(String.format("Handle request %s %n", context.getRequestId()));
```
Comment (shuaichang) [inline README.md]: \n?

File: fc-java-core/src/main/java/com/aliyun/fc/runtime/Context.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package com.aliyun.fc.runtime;
+
+/**
+ * The context object allows you to access useful information available within the alibaba cloud
+ * function computing execution environment
```
Comment (shuaichang) [inline fc-java-core/src/main/java/com/aliyun/fc/runtime/Context.java]: function compute?

File: fc-java-core/src/main/java/com/aliyun/fc/runtime/FunctionParam.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    /**
+     * Gets the execution time limit of the function
+     */
+    public int getExecTimeLimitInMillis();
+
+    /**
+     * Gets the maximum memory size configured for the function
+     */
+    public int getMemoryLimitInMB();
+
```
Comment (shuaichang) [inline fc-java-core/src/main/java/com/aliyun/fc/runtime/FunctionParam.java:27]: Remove blank line

Comment (nerdyyatrice) [reply fc-java-core/src/main/java/com/aliyun/fc/runtime/FunctionParam.java]: Actually, that seems to be the convention for all interfaces. I checked the Lambda's interface.

**Review by shuaichang — COMMENTED** [review]
LGTM

**Pull request merged**